### PR TITLE
chore(updatecli): more information to retrieve credential within terraform

### PR DESCRIPTION
**Pull request by smerle33**
following #712 more informations

### CODE REVIEW

File: updatecli/updatecli.d/trusted.ci.sp.endate.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ actions:
       title: Generate a new Azure AD Application password with expiration date {{ source "nextExpiry" }} on `trusted.ci.jenkins.io`
       description: |
         This PR generates a new Azure AD application password with a new end date for the `trusted.ci.jenkins.io` controller (to allow spawning Azure VM agents).
-        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password's value in Terraform state (do NOT save it anywhere!) and update (manually) the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
+        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password's value in Terraform state [terraform show -json / module.trusted_ci_jenkins_io.azuread_application_password.controller:values.value] (do NOT save it anywhere!) and update (manually) the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
```
Comment (lemeurherve) [inline updatecli/updatecli.d/trusted.ci.sp.endate.yaml:58]:
```suggestion
        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password value from the Terraform state with `terraform show -json / module.trusted_ci_jenkins_io.azuread_application_password.controller:values.value` (do NOT save it anywhere!) and (manually) update the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
```

What's the `/` for?

**Review by lemeurherve — DISMISSED** [review]

File: updatecli/updatecli.d/trusted.ci.sp.endate.yaml [hunk 2]
```diff
@@ -55,7 +55,7 @@ actions:
...
-        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password's value in Terraform state (do NOT save it anywhere!) and update (manually) the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
-        Finally, verify both the Azure Credential and Azure VM clouds by clicking on the "Verify <...>" buttons with success.
+        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password value from the Terraform state with `terraform show -json / module.trusted_ci_jenkins_io.azuread_application_password.controller:values.value` (do NOT save it anywhere!) and (manually) update the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
+        Finally, verify both the Azure Credential and Azure VM clouds by clicking on the "Verify <...>" buttons with success, restart the controller to ensure that the old credential is not kept in cache.
```
Comment (lemeurherve) [inline updatecli/updatecli.d/trusted.ci.sp.endate.yaml:59]:
```suggestion
        Finally, verify both Azure Credential and Azure VM clouds by checking that a click on the "Verify <...>" buttons returns a success, then restart the controller to ensure that the old credential is not kept in cache.
```

**Review by lemeurherve — APPROVED** [review]

File: updatecli/updatecli.d/trusted.ci.sp.endate.yaml (hunk 1 above)
Comment (smerle33) [reply updatecli/updatecli.d/trusted.ci.sp.endate.yaml]: as explain in other channel, just a way to identify the command versus the text I think ...

File: updatecli/updatecli.d/trusted.ci.sp.endate.yaml [hunk 3]
```diff
@@ -55,7 +55,7 @@ actions:
       title: Generate a new Azure AD Application password with expiration date {{ source "nextExpiry" }} on `trusted.ci.jenkins.io`
       description: |
         This PR generates a new Azure AD application password with a new end date for the `trusted.ci.jenkins.io` controller (to allow spawning Azure VM agents).
-        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password's value in Terraform state (do NOT save it anywhere!) and update (manually) the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
-        Finally, verify both the Azure Credential and Azure VM clouds by clicking on the "Verify <...>" buttons with success.
+        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password value from the Terraform state with `terraform show -json / module.trusted_ci_jenkins_io.azuread_application_password.controller:values.value` (do NOT save it anywhere!) and (manually) update the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
```
Comment (lemeurherve) [inline updatecli/updatecli.d/trusted.ci.sp.endate.yaml:58]:
```suggestion
        Once this PR is merged and deployed with success by Terraform (on infra.ci.jenkins.io), you can retrieve the new password value from the Terraform state with `terraform show -json` then searching for `values.value` in the `module.trusted_ci_jenkins_io.azuread_application_password.controller` section (do NOT save it anywhere!) and (manually) update the trusted.ci.jenkins.io credential named `azure-jenkins-sponsorship-credentials` through the Jenkins UI.
```

Comment (smerle33) [reply updatecli/updatecli.d/trusted.ci.sp.endate.yaml]: Thanks !

**Review by dduportal — APPROVED** [review]

**Pull request merged**